Metadata panel

chimosky opened the pull request:
I've made some changes to the metadata panel, and I'd like to know if I'm headed in the right direction and how to proceed.

@romicolman kindly review, this is from #727 and I was told you'll help point me in the right direction.

---

Please make sure that all the checks pass. Please add here any additional information regarding this pull request. It's highly recommended that you link this PR to an issue (please create one if it doesn't exist for this PR)

romicolman commented:
HI @chimosky! First of all, thank you very much for contributing to the ODE. It's great to see that the ODE is getting contributions!

I managed to see your implementations. A couple of comments based on what I see:

- I use Mac. I'm not able to see all fields in my screen. Here is a screenshot of my current view:

<img width="1091" alt="Captura de pantalla 2025-03-05 a las 14 18 38" src="https://github.com/user-attachments/assets/1d5cab9b-6bfe-48c8-89f6-595cccd10c1e" />

**Idea:** distribute fields across the screen as we did for the stable version:

<img width="1102" alt="Captura de pantalla 2025-03-05 a las 14 27 29" src="https://github.com/user-attachments/assets/7e196d5d-f7f4-439b-902e-4f65d813c520" />

- We would like to keep all sections and metadata categories on the left side. We know that the stable version had the main menu at the top, but for the migrated one we want to follow the "tree logic" that we have on the sidebar.
- **Resource section:** I'm not able to see fields under the Source and Contributors categories.
- **Dialect:** default fields are missing. Same for the Csv category
- **Schema:** I'm not able to edit the default fields (name, primary key, title, etc.).

Additional comments

If possible, could you remove the Metadata term serving as the section title? 
<img width="1133" alt="Captura de pantalla 2025-03-05 a las 14 41 36" src="https://github.com/user-attachments/assets/7ff4b191-426b-4c38-850a-7cb3b1409f7e" />

**General style:** can we apply to the metadata panel the same font and style that we have for the rest of the application?

Thank you so much for collaborating with the ODE, @chimosky !

chimosky commented:
@romicolman I've addressed some of the issues you mentioned, kindly test.

I'm also trying to resolve the issue of applying the same font and style to the metadata panel, I'll get to it once I can.
